k2/python/k2/fsa_algo.py

```diff
@@ -1069,3 +1072,58 @@ def ctc_topo(max_token: int,
     ragged_arc, aux_labels = _k2.ctc_topo(max_token, device, modified)
     fsa = Fsa(ragged_arc, aux_labels=aux_labels)
     return fsa
+
+def levenshtein_graph(symbols: Union[k2.RaggedTensor, List[List[int]]],
+        self_loop_weight: float = -1,
```

Comment:
@danpovey What would you suggest to name this argument, `self_loop_weight` may not be a good one.

Reply:
It represents insertion and deletion cost. How about calling it "ins_del_cost"?

Reply:
How about naming it `penalty`, and the fsa's attribute named `penalty_bias`.